web-animations-1/Overview.bs

```diff
@@ -4518,8 +4527,9 @@ dictionary OptionalEffectTiming {
     time taken to complete a single iteration of the <a>animation
     effect</a>.
 
-    In this level of this specification, the string value <code>auto</code>
-    is treated as the value zero for the purpose of timing model calculations
+    The string value <code>auto</code> is treated as the
```

Comment:
Aww, GH does not allow me to comment on other lines that is not in your change 👎 

But a few lines below there is 
```
This is a forwards-compatiblity measure since a future level of this
    specification is expected to introduce group effects where the
    <code>auto</code> value expands to include the duration of the child
    effects.

````

With your change where auto is taken to mean timeline's duration, is it possible to have the flexibility to interpret it differently?
e.g., prefer's child effect effect durations if timeline duration is infinite?

Reply:
My thinking was actually if this was a group effect auto would look at the children's timing, add up all of the non-auto children timings and distribute the remaining time on the timeline between the auto children. i.e.
Having children of durations [auto, 12s, 8, auto] on a 100s ScrollTimeline would turn into [40s, 12s, 8s, 40s]

In this manner it can always be seen to be looking at the children, but if there are no children it is distributing remaining time from the timeline.

Reply:
I added a few comments similar to this in #4907. It would be nice to get a consistent approach that covers all these three cases: scroll timelines, group effects, and keyframe effects with time-based keyframe offsets.